### ISSUE TITLE
how to delete the stack

### ISSUE THREAD

**Issue by dharmendrakariya** [plippe/aws-lambda-ip-availability#5]
Hey there, 

Thanks for the good work.

I want to test but just a question that how do I destroy the stack once I am done.

**Comment by plippe** [comment]
Hi @dharmendrakariya ,

I just saw your issue today, so sorry about that

In this project, I use `npm` to hide the complexity of my entry points. While I documented a few in the README.md, all of them are in the package.json file.

```
  "scripts": {
    "format": "...",
    "package": "...",
    "publish": "...",
    "create-cloudformation-stack": "...",
    "update-cloudformation-stack": "...",
    "delete-cloudformation-stack": "..."
  }
```

There are 3 methods related to the stack, `create-cloudformation-stack`, `update-cloudformation-stack, `delete-cloudformation-stack`.

Running `npm run delete-cloudformation-stack` should delete the stack.
